### PR TITLE
Updating security issue, fixing versions in pom fixes #3

### LINKED ISSUE #3: patch for security alert from github
Dependencyorg.apache.httpcomponents:httpclientVersion< 4.3.6Upgrade to~> 4.3.6VulnerabilitiesCVE-2015-5262 Moderate severityDefined inpom.xml | Dependencyorg.apache.httpcomponents:httpclientVersion< 4.3.6Upgrade to~> 4.3.6VulnerabilitiesCVE-2015-5262 Moderate severityDefined inpom.xml | Dependencyorg.apache.httpcomponents:httpclient | Version< 4.3.6 | Upgrade to~> 4.3.6 | VulnerabilitiesCVE-2015-5262 Moderate severity | CVE-2015-5262 Moderate severity | Defined inpom.xml
-- | -- | -- | -- | -- | -- | -- | --
Dependencyorg.apache.httpcomponents:httpclientVersion< 4.3.6Upgrade to~> 4.3.6VulnerabilitiesCVE-2015-5262 Moderate severityDefined inpom.xml | Dependencyorg.apache.httpcomponents:httpclient | Version< 4.3.6 | Upgrade to~> 4.3.6 | VulnerabilitiesCVE-2015-5262 Moderate severity | CVE-2015-5262 Moderate severity | Defined inpom.xml
Dependencyorg.apache.httpcomponents:httpclient | Version< 4.3.6 | Upgrade to~> 4.3.6
VulnerabilitiesCVE-2015-5262 Moderate severity | CVE-2015-5262 Moderate severity | Defined inpom.xml
CVE-2015-5262 Moderate severity

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Comment by jonespm** [comment]
Where were you releasing this at? We could relatively easily release it to maven central if this groupId were correct (fixed here). Since I don't *really* want to install maven and java locally I updated the readme to be able to compile without installing that. Looks like it compiles fine.

**Comment by jonespm** [comment]
This is a dependency for a number of our projects internally so this should be fixed.

File: pom.xml [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>lti</groupId>
+  <groupId>edu.umich.it.tl</groupId>
```
Comment (zqian) [inline pom.xml:5]: With the groupid change, we need to update pom files that depend on lti-utils

File: README.md [hunk 2]
```diff
@@ -83,3 +83,14 @@ the following in your catalina.out log when you press "Launch" (jsfiddle) or
 
    MyRequest: true
 
+[Docker based compilation]
```
Comment (zqian) [inline README.md:86]: nice docker support!

**Review by zqian — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (jonespm) [reply pom.xml:5]: Yeah, I figured we'll have to update this version anyway anywhere that uses it. And if we have a correct groupId we can just release it to Maven Central saving some work down the road.

File: README.md (hunk 2 above)
Comment (jonespm) [reply README.md:86]: Yeah, I *really* like compiling java projects via docker. Avoids having to install anything locally, allows anyone to run it and keeps the versions fixed. Could probably put this as a shell script locally too but easy to copy/paste for the moment.

File: pom.xml (hunk 1 above)
Comment (pushyamig) [reply pom.xml:5]: The projects that are currently using this is CCM and Gdrive. I will test this fix with the CCM as it need a release soon for the infrastructure changes

Comment (jonespm) [reply pom.xml:5]: Thanks! I put this change on CCM at https://github.com/tl-its-umich-edu/canvas_course_manager/pull/249

I tested that the Dockerfile builds

**Pull request merged**